### ISSUE TITLE
Add custom color for direct messages

### ISSUE THREAD

**Issue by TheRealKingS** [syuilo/Misskey#8585]
## Summary

A custom color, or even custom CSS for direct messages would be awesome. In the normal timeline these are otherwise often missed, even if there is a notification under More.

**Comment by Johann150** [comment]
I don't think it is intended primarily to view direct messages in the timeline, that is what the direct notes menu entry is for. You can customize your menu bar to add it, I think it even shows if there are unread ones.
